client/src/components/PrivateRoute.tsx

```diff
@@ -0,0 +1,15 @@
+import { Navigate, useLocation } from 'react-router-dom';
+import { useAuth } from '@context/useAuth';
```

Comment:
Let's move this useAuth to hooks **folder**, not **context** one, I think is more appropriated.

Reply:
moved